lib/src/presentation/widgets/questionnaire_item/base/questionnaire_multi_choice_item_view.dart

```diff
@@ -51,7 +51,11 @@ abstract class QuestionnaireMultiChoiceItemViewState<
     } else {
       selectedValues.remove(value);
     }
-    setState(() {});
+
+    setState(() {
```

Comment:
this is not necessary as the `selectedValues` is a `get` property of the `controller.value` itself